playground/editors/UVEditor.js

```diff
@@ -12,7 +12,9 @@ export class UVEditor extends BaseNodeEditor {
 
 		this.setOutputLength( 2 );
 
-		const optionsField = new SelectInput( [ '1', '2' ], 0 ).onChange( () => {
+		const options = Array.from( Array( 4 ).keys() ).map( String );
```

Comment:
I think `[ '0', '1', '2', '3' ]` is much more readable.

Reply:
I agree, I was supposed to be customized, but I ended up removing that option.